spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/AnnotatedServiceRegistrationBean.java

```diff
@@ -33,7 +38,10 @@
  * >             .setServiceName("myAnnotatedService")
  * >             .setPathPrefix("/my_service")
  * >             .setService(new MyAnnotatedService())
- * >             .setDecorator(LoggingService.newDecorator());
+ * >             .setDecorator(LoggingService.newDecorator())
+ * >             .setExceptionHandlers(ImmutableList.of(new MyExceptionHandler()))
+ * >             .setRequestConverters(ImmutableList.of(new MyRequestConverter()))
+ * >             .setResponseConverters(ImmutableList.of(new MyResponseConverter()));
```

Comment:
- Could we also have the versions of these methods that accept varargs?
- We could define some common supertype of all these registration beans to reduce code duplication.

Reply:
Added varargs versions.

Reply:
Then perhaps we could update this example for less verbosity?